server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -440,6 +441,14 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
         new CompactionJobGenerator(new ServiceEnvironmentImpl(manager.getContext()),
             tableMgmtParams.getCompactionHints(), tableMgmtParams.getSteadyTime());
 
+    try {
+      CheckCompactionConfig.validate(manager.getConfiguration());
+    } catch (SecurityException | IllegalArgumentException | IllegalStateException
+        | ReflectiveOperationException e) {
+      LOG.error("Error validating compaction configuration, not performing compactions.", e);
```

Comment:
Pausing compaction seems like the correct action to take since compaction starvation of a particular table could occur if two different planners are sending different priority jobs to the same queue. 

I feel like this message needs stronger wording 
`Error validating compaction configuration, all compactions are paused until the config is fixed`

Reply:
Updated in c561adc